### DISCUSSION TITLE
Transition jumpiness

### DISCUSSION THREAD

**Discussion by djglxxii** [alpinejs/alpine#1229] [1. Help]
How can I make my Alpine transitions not jump when elements are removed then added?

Here is a code pen example of what I'm experiencing:

https://codepen.io/djglxxii-the-sasster/pen/dyNWjwZ

Notice how when you go from selection to selection, the elements fade out/in as expected but abruptly move up to fill the gap left by the removed elements.

**Comment by KevinBatdorf** [comment]
I think you need to use some creative CSS positioning to achieve what you want. Or maybe someone has figured out a good solution already.

Alpine doesn't have an in-out transition type like Vue does.

**Comment by aaronbushnell** [comment]
I'm seeing this as well @djglxxii and I setup a comparison of V2's behavior to V3 in some GIFs that hopefully help demonstrate this! #1561

**Comment by calebporzio** [comment]
Here's the fix: https://github.com/alpinejs/alpine/pull/1566

Will be in next release. Thanks!
